ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.11.yaml

```diff
@@ -60,23 +63,15 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-f14
-  cron: 19 16 2,16 * *
+- as: aws-ipi-baselinecaps-none-f14
```

Comment:
maybe we need to wait for the similar confirm as https://github.com/openshift/release/pull/46102#discussion_r1408906877

Reply:
Ah, good catch! It is what we are going to discuss in tomorrow's meeting. Dynamically check if a case is appropriate to run in the automation or relying on the job owner to control it.

Reply:
For 4.11, the optional caps is `v411="baremetal marketplace openshift-samples"`, I guess we do not exclude the cases;
For 4.12, we need to exclude storage cases, because `v412=" ${v411} Console Insights Storage CSISnapshot"`
For 4.13, we should be equal to 4.12, because `v413=" ${v412} NodeTuning"`

Reply:
We only need to exclude v411="baremetal marketplace openshift-samples". Right?

Reply:
How do I exclude the test-cases in baremetal marketplace openshift-samples? @liangxia we need your help.

Reply:
AFAIK, `openshift-samples` provides some image streams and templates which can be consumed by `oc new-app` related commands. It's hard to say which teams test cases can be or can not be run with/without it.
`marketplace` enables the ability to install optional operators from OperatorHub, which is hard to say which team is blocked or not as it depends on whether the team use/test optional operators.
For `baremetal`, I'm not exactly sure what's its meaning there and what it affects.